docs/link_checker_api_integration.md

```diff
@@ -0,0 +1,36 @@
+# Link Checker API Integrations
+
+Links on an edition are checked using the LinkChecker API. Links are checked when the `Check Links` button is clicked on the edition show page. This fires off a request to the Link Checker API using the `/create_batch` endpoint. Once this has checked all the links present in the edition it makes a callback with a `batch_report` that gets saved as a `LinkCheckerApiReport`.
+
+## LinkCheckerApiReports / LinkCheckerApiReportLinks
+
+| Field                 |
```

Comment:
I don't think it's necessary to list the fields here, just a few sentences explaining each model and what it's used for.